src/components/Checkbox/README.md

```diff
@@ -31,13 +31,13 @@ In order to override the theme, use Wix Stylable extends capabilities and wrap w
 2. Create a component that uses it
     ``` javascript
     import Checkbox from 'wix-ui-tpa/Checkbox';
-    import checkboxStylesExt from './CheckboxExt.st.css';
+    import { st, classes } from './CheckboxExt.st.css';
 
     const CheckboxExt = (props) => (
         <Checkbox
             onChange={() => {}}
             label="Label"
+            className={st(classes.root)} 
             {...props}
```

Comment:
```suggestion
            {...props}
            className={st(classes.root, props.className)}
```

Reply:
again, an example of extending a component